--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,10 +11,8 @@ endif()
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 include(third-party)
-get_third_party(glog)
+find_package(glog REQUIRED)
 set(WITH_GFLAGS OFF CACHE INTERNAL "" FORCE)
-add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/glog ${CMAKE_CURRENT_BINARY_DIR}/glog EXCLUDE_FROM_ALL)
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/glog ${CMAKE_CURRENT_BINARY_DIR}/glog/src)
 add_definitions(-DC10_USE_GLOG=1)
 
 add_library(
