--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ nix = { workspace = true, optional = true }
 # For power status with upower feature
 upower_dbus = { git = "https://github.com/pop-os/dbus-settings-bindings", rev = "badfc6a", optional = true }
 # Required for some features
-zbus = { workspace = true, optional = true }
+zbus = { workspace = true }
 # Internationalization
 i18n-embed = { version = "0.14", features = [
     "fluent-system",
@@ -50,7 +50,7 @@ default = ["logind", "networkmanager", "upower"]
 logind = ["logind-zbus", "zbus"]
 networkmanager = ["cosmic-dbus-networkmanager", "zbus"]
 upower = ["upower_dbus", "zbus"]
-zbus = ["dep:zbus", "nix"]
+zbus = ["nix"]
 
 [workspace]
 members = ["daemon"]
